pkg/client/s3client.go

```diff
@@ -118,7 +118,7 @@ func NewS3Client(c *Client, instanceName, region string) (s3client *S3Client, er
 func (c *S3Client) CheckBucketExists(bucketName string) (bool, error) {
 	result, err := c.S3Session.ListBuckets(nil)
 	if err != nil {
-		klog.Infof("Unable to list buckets, %v\n", err)
+		klog.V(1).Infof("Unable to list buckets, %v\n", err)
```

Comment:
```suggestion
		klog.V(1).Infof("Unable to list buckets, %v", err)
```
Ref: https://pkg.go.dev/k8s.io/klog#Infof

Reply:
Done.